libs/seo/src/lib/directives/meta-property.directive.ts

```diff
@@ -0,0 +1,83 @@
+import { AfterContentChecked, Directive, ElementRef, inject, Input, OnDestroy, OnInit } from '@angular/core';
+
+import { MetaId, SEO_OPTIONS } from '../models';
+import { MetaValueCmd, SeoService } from '../services';
+
+@Directive({
+  selector: '[czMetaProperty]',
+  standalone: true,
+})
+export class MetaPropertyDirective implements AfterContentChecked, OnDestroy, OnInit {
+  @Input('czMetaProperty')
+  metaId?: MetaId;
+
+  @Input('czMetaPropertySubValuesSeparator')
+  set subValuesSeparator(value: string | undefined) {
+    this._subValuesSeparator = value;
+  }
+
+  get subValuesSeparator(): string | undefined {
+    return this._subValuesSeparator ?? this.parent?.subValuesSeparator ?? this.seoOptions.metaPropertyDirective?.subValuesSeparator;
+  }
+
+  @Input('czMetaPropertyWatchForChanges')
+  public get watchForChanges(): boolean | undefined {
+    return this._watchForChanges ?? this.parent?.watchForChanges ?? this.seoOptions.metaPropertyDirective?.watchForChanges;
+  }
+  public set watchForChanges(value: boolean | undefined) {
+    this._watchForChanges = value;
+  }
+
+  children: MetaPropertyDirective[] = [];
+
+  private firstChange = true;
+  private lastContent = '';
+  private cmd?: MetaValueCmd;
+  private _subValuesSeparator?: string;
+  private _watchForChanges?: boolean | undefined;
+  private seoOptions = inject(SEO_OPTIONS);
+  private elRef = inject<ElementRef<HTMLElement>>(ElementRef);
+  private parent? = inject(MetaPropertyDirective, { skipSelf: true, optional: true });
+  private seoService = inject(SeoService);
+
+  ngOnInit(): void {
+    this.parent?.registerChild(this);
+  }
+
+  ngAfterContentChecked(): void {
+    if (this.parent || (!this.watchForChanges && !this.firstChange) || !this.metaId) return;
+    this.firstChange = false;
+    const content = this.getContent();
+    if (content === this.lastContent) return;
```

Comment:
Computing content is "costly" since we are in `ngAfterContentChecked`, which is ran a lot. So I prefer to do compute as late as possible. But I will move `this.firstChange = false` afterward, it makes more sense.